ql/CMakeLists.txt

```diff
@@ -1,2 +1,9 @@
 file(GLOB_RECURSE QUANTLIB_FILES "*.hpp" "*.cpp")
 add_library(QuantLib SHARED ${QUANTLIB_FILES})
+
+install(DIRECTORY . DESTINATION include/QuantLib
```

Comment:
Does this install, for example, as `/usr/local/include/QuantLib/ql/version.hpp`? I'd follow what happens with automake and install to `include/ql/version.hpp` instead.